circle.yml

```diff
@@ -6,6 +6,6 @@ test:
   pre:
     - npm run eslint
   override:
-    - npm run istanbul
+    - npm test
```

Comment:
This will mean we are running `eslint` twice. e.g. in the `pre` section and as part of `npm test`. I kind of like separating the test stages in CircleCI for clarity as to which stage failed, but I'm open to just keeping `npm test`. IIRC the `override` section can be removed if we are using `npm test`.